Source/Core/Core/HW/WiimoteReal/WiimoteReal.cpp

```diff
@@ -669,15 +669,19 @@ void WiimoteScanner::ThreadFunc()
   {
     m_scan_mode_changed_event.WaitFor(std::chrono::milliseconds(500));
 
+    if (m_scan_mode.load() == WiimoteScanMode::DO_NOT_SCAN)
+      continue;
+
+    // Stop scanning if not in continous mode.
+    auto scan_mode = WiimoteScanMode::SCAN_ONCE;
+    m_scan_mode.compare_exchange_strong(scan_mode, WiimoteScanMode::DO_NOT_SCAN);
+
     // Does stuff needed to detect disconnects on Windows
     for (const auto& backend : m_backends)
       backend->Update();
 
     CheckForDisconnectedWiimotes();
```

Comment:
I'm worried that not calling `Update` and `CheckForDisconnectedWiimotes` regularly will be a problem (especially on Windows).

Were these functions doing something undesirable on your system when "Continuous Scanning" was off?

Reply:
Was this issue fixed at all?

My issues with this started with this commit https://github.com/dolphin-emu/dolphin/pull/8575

On previous builds I just could turn off continuous scan and desire effect will be fine.

But after that commit, or continuous scan is somehow always doing something, or dolphin started to reset bt adapter status even when having continuous scan disabled. Which didn't happen before that commit.